### PR TITLE
Add abstract to PackageInfo.g

### PR DESCRIPTION
**Pull request by mtorpey**
This would resolve Issue #39.

### CODE REVIEW

File: PackageInfo.g [hunk 1]
```diff
@@ -58,7 +58,11 @@ ArchiveFormats := ".tar.gz",
 ##
 Status := "dev",
 
-AbstractHTML := "",
+AbstractHTML :=
+  "<span class=\"pkgname\">PackageManager</span> is a basic collection of \
+   simple functions for installing and removing \
+   <span class=\"pkgname\">GAP</span> packages, with the eventual aim of \
+   becoming a full pip-style package manager for the GAP system.",
```
Comment (alex-konovalov) [inline PackageInfo.g]: The second `GAP` should also be a `<span>` element.

Comment (mtorpey) [reply PackageInfo.g]: I've added a new commit to fix this.

**Comment by mtorpey** [comment]
How's that? :)

**Comment by mtorpey** [comment]
Thanks!

**Pull request merged**